cards.tex

```diff
@@ -46,6 +46,22 @@
 \newcommand{\Reference}[2]{{\small \color{gray} \engschrift #1 #2}}
 \newcommand{\DamageType}[1]{\operatorname{\mbox{#1}}}
 
+\newcommand{\Tag}[2]{%
+    \tikz[baseline]{%
+        \node[anchor=base, text=white, fill=#1, font=\sffamily, text depth=.5mm] {#2};
+    }%
+}
+
+\definecolor{tagsBg}{RGB}{217, 196, 132}
+\definecolor{uncommonBg}{RGB}{152, 81, 61}
+\definecolor{rareBg}{RGB}{0, 38, 100}
+\definecolor{traitBg}{RGB}{94, 0, 0}
+
+\newcommand{\Trait}[1]{\Tag{traitBg}{#1}}
+\newcommand{\Rare}[0]{\Tag{rareBg}{Rare}}
+\newcommand{\Uncommon}[0]{\Tag{uncommonBg}{Uncommon}}
```

Comment:
`Rare` and `Uncommon` are helpful when looking in the books (because players need to ask the GM whether they can get these in a particular shop). But are they important on the item card when the player already acquired it?

Reply:
Do they affect costs or difficulty for _repairing_ the item? 

Is the information helpful to balance the game (for players and / or the GM) by knowing how many items from which category the party currently has?

Also, the cards might be used as a "shop front", and this makes it easier to draft a deck with say 1 rare, 3 uncommon, and _n_ common items...

Reply:
Ok, convinced :)